app/models/launch_job_task.rb

```diff
@@ -0,0 +1,34 @@
+class LaunchJobTask < MqttClientTask
+  def service_options
```

Comment:
are these options sent to mqtt_client? if yes, better directly switch to use tower terms now.